problems/6.random.js

```diff
@@ -24,8 +24,8 @@ function diceRoll() {
  * @param {number} n - A number that serves as the upper limit of random
  * @returns {number} - A random number betweeen 0 and n
  */
-function randRange() {
-
+function randRange(n) {
+  return Math.floor(Math.random() * n) + 1;
```

Comment:
by adding one here you'll never get 0 which should be an option here. The idea of adding one is correct but this is in the wrong place